roles/nginxplus/files/conf/http/dss-prod.conf

```diff
@@ -11,12 +11,20 @@ limit_req_zone $external_traffic zone=dss-prod-ratelimit:10m rate=10r/s;
 
 upstream dss-prod {
     zone dss-prod 64k;
-    server dss-prod1.princeton.edu resolve max_fails=0;
-    server dss-prod2.princeton.edu resolve max_fails=3;
+    # lowchallenge challenges after 20 requests from xx.xx.<>.<>.<>
+    server service.consul service=lowchallenge.traefik-wall-production resolve max_fails=0;
+    # highchallenge challenges everyone. Comment above and uncomment
+    # this when you're being swarmed by bots.
+    # server service.consul service=highchallenge.traefik-wall-staging resolve max_fails=0;
+    # If Traefik crashes, fall down to the box directly.
+    server dss-prod1.princeton.edu max_fails=0 resolve backup;
+    server dss-prod2.princeton.edu max_fails=0 resolve backup;
     sticky learn
           create=$upstream_cookie_dssprodcookie
           lookup=$cookie_dssprodcookie
           zone=dssprodclient_sessions:1m;
+    resolver nomad-host-prod1.lib.princeton.edu nomad-host-prod2.lib.princeton.edu nomad-host-prod3.lib.princeton.edu valid=5s;
+    resolver_timeout 2s;
 }
 
```

Comment:
You'll need the extra headers like in https://github.com/pulibrary/princeton_ansible/blob/main/roles/nginxplus/files/conf/http/dpul-prod.conf#L49-L50.